package.json

```diff
@@ -44,22 +42,20 @@
   "devDependencies": {
     "@babel/core": "^7.23.5",
     "@babel/eslint-parser": "^7.23.3",
-    "@babel/plugin-syntax-dynamic-import": "^7.8.3",
     "@babel/preset-env": "^7.23.5",
     "@babel/preset-react": "^7.23.3",
     "@typescript-eslint/eslint-plugin": "^6.12.0",
     "@typescript-eslint/parser": "^6.12.0",
-    "babel-loader": "^8.2.3",
-    "css-loader": "^2.1.0",
+    "babel-loader": "^9.2.1",
+    "css-loader": "^7.1.2",
     "ejs": "^3.1.10",
-    "eslint": "^8.57.1",
+    "eslint": "^8.57.0",
```

Comment:
https://github.com/TerriaJS/terriajs/pull/7351 bumps that ESLint to 8.57.1, so I think this change is accidental?